tst/EnergyPlus/unit/PlantLoopHeatPumpEIR.unit.cc

```diff
@@ -5393,5 +5394,399 @@ TEST_F(EnergyPlusFixture, CoolingSimulate_WSHP_SourceSideOutletTemp)
+        "  auxElecEIRFoPLRCurve,                   !- Auxiliary Electric Energy Input Ratio Function of PLR Curve Name",
+        "  20;                                     !- Standby Electric Power {W}",
+
+        "OutdoorAir:Node,",
+        "  FuelFired GAHP Heating OA Node;         !- Name",
+
+        "HeatPump:AirToWater:FuelFired:Cooling,",
+        "  FuelFired GAHP Cooling,                 !- Name",
+        "  FuelFired GAHP Cooling Water Inlet Node, !- Water Inlet Node Name",
+        "  FuelFired GAHP Cooling Water Outlet Node, !- Water Outlet Node Name",
+        "  FuelFired GAHP Cooling OA Node,         !- Air Source Node Name",
+        "  FuelFired GAHP Heating,                 !- Companion Heating Heat Pump Name",
+        "  NaturalGas,                             !- Fuel Type",
+        "  GAHP,                                   !- End-Use Subcategory",
+        "  4000,                                   !- Nominal Cooling Capacity {W}",
+        "  2,                                      !- Nominal COP {W/W}",
+        "  0.006,                                  !- Design Flow Rate {m3/s}",
+        "  7,                                      !- Design Supply Temperature {C}",
+        "  11.1,                                   !- Design Temperature Lift {deltaC}",
+        "  1,                                      !- Sizing Factor",
+        "  NotModulated,                           !- Flow Mode",
+        "  DryBulb,                                !- Outdoor Air Temperature Curve Input Variable",
+        "  EnteringEvaporator,                     !- Water Temperature Curve Input Variable",
+        "  CapCurveFuncTemp,                       !- Normalized Capacity Function of Temperature Curve Name",
+        "  EIRCurveFuncTemp,                       !- Fuel Energy Input Ratio Function of Temperature Curve Name",
+        "  EIRCurveFuncPLR,                        !- Fuel Energy Input Ratio Function of PLR Curve Name",
+        "  0.1,                                    !- Minimum Part Load Ratio",
+        "  1,                                      !- Maximum Part Load Ratio",
+        "  CRFCurve,                               !- Cycling Ratio Factor Curve Name",
+        "  500,                                    !- Nominal Auxiliary Electric Power {W}",
+        "  auxElecEIRCurveFuncTempCurve,           !- Auxiliary Electric Energy Input Ratio Function of Temperature Curve Name",
+        "  auxElecEIRFoPLRCurve,                   !- Auxiliary Electric Energy Input Ratio Function of PLR Curve Name",
+        "  20;                                     !- Standby Electric Power {W}",
+
+        "OutdoorAir:Node,",
+        "  FuelFired GAHP Cooling OA Node;         !- Name",
+
+        "Curve:Biquadratic,",
+        "  CapCurveFuncTemp,                       !- Name",
+        "  1,                                      !- Coefficient1 Constant",
+        "  0,                                      !- Coefficient2 x",
```

Comment:
It would be nice if these curves actually computed a value instead of a constant 1.0. The constant fT curves in the example file explain why there are no diffs. That would be nice to change, but not sure where to get a reasonable set of curves.